go/src/koding/kites/kloud/kloud/kloud.go

```diff
@@ -30,6 +31,7 @@ import (
 	"koding/kites/kloud/terraformer"
 	"koding/kites/kloud/userdata"
 	"koding/remoteapi"
+	"koding/socialapi"
```

Comment:
Done in https://github.com/koding/koding/pull/9870/commits/4ff0b9494a13d1dfaf3680748d4209c044bc6646.

Renamed to`api`. Note for future - we're going to merge the following packages:

1) `koding/klientctl/endpoint`
2) `koding/remoteapi`

Under `koding/api` in future. For now we're not done with 1) API-wise so we do not see yet how would the final result look like. Just fyi.